docs/guide/options.md

```diff
@@ -53,21 +53,35 @@ const chart = new Chart('id', {
 
 | Name | Type | Default | Description
 | ---- | ---- | ------- | ----------
-| `enabled` | `boolean` | `false` | Enable zooming
-| `drag` | `boolean`\|[`DragEffectOptions`](#drag-effect-options) | `undefined` | Enable drag-to-zoom behavior (disables zooming by wheel)
+| `wheel` | [`WheelOptions`](#wheel-options) | `undefined` | Options of the mouse wheel behavior
```

Comment:
Should make sure this is in the release notes as a breaking change. Or we can keep `enabled` at the top level and do some kind of fallback

Reply:
I think it's the good moment to make breaking changes since the final 1.0.0 has not been released yet (I guess there were other breaking changes in previous beta releases but I'm not sure)? I'm afraid that supporting a fallback could bring some confusion when a specific behavior works without having been explicitly enabled?

+ the warning in the console will make migration easier for users who didn't read the release notes. 

But yeah, I agree that a mention in the release note is really important. I can edit the release draft if you want.

Reply:
@jledentu just in case you hadn't noticed, release drafter rewrites the notes after every commit to master, so should do the edit after the last commit (bump) before release.